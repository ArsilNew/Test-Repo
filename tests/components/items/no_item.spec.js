import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import NoItems from "../../../src/components/items/no_items";

describe("NoItem", () => {
  test("default snapshot", () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<NoItems />);
    expect(tree).toMatchSnapshot();
  });
});
