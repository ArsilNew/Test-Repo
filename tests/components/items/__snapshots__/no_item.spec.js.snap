// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoItem default snapshot 1`] = `
<div
  className="no_items"
>
  No Items...
</div>
`;
